package.json

```diff
@@ -223,7 +223,7 @@
     "stylelint-checkstyle-formatter": "^0.1.2",
     "stylelint-scss": "^5.3.2",
     "terser-webpack-plugin": "^5.2.4",
-    "testcafe": "^1.20.1",
+    "testcafe": "^3.7.0-rc.2",
```

Comment:
Do we have to use an `-rc` version?

Reply:
For Bacon we can use latest stable `3.6.2`, but locally it will not work with latest Chrome 130. 
`3.7.0-rc.2` fixes issue with Chrome 130